lib/Swagger/src/TemplateClassification/Storage/Api/TCSApi.php

```diff
@@ -44,83 +51,84 @@
  * @category Class
  * @package  Swagger\Client
  * @author   http://github.com/swagger-api/swagger-codegen
- * @license  http://www.apache.org/licenses/LICENSE-2.0 Apache Licene v2
+ * @license  http://www.apache.org/licenses/LICENSE-2.0 Apache License v2
  * @link     https://github.com/swagger-api/swagger-codegen
  */
 class TCSApi
 {
-
     /**
      * API Client
+     *
      * @var \Swagger\Client\ApiClient instance of the ApiClient
      */
     protected $apiClient;
-  
+
     /**
      * Constructor
+     *
      * @param \Swagger\Client\ApiClient|null $apiClient The api client to use
      */
-    function __construct($apiClient = null)
+    public function __construct(\Swagger\Client\ApiClient $apiClient = null)
     {
-        if ($apiClient == null) {
+        if ($apiClient === null) {
             $apiClient = new ApiClient();
-            $apiClient->getConfig()->setHost('https://localhost/');
+            $apiClient->getConfig()->setHost('https://localhost/template-classification-storage');
         }
-  
+
         $this->apiClient = $apiClient;
     }
-  
+
     /**
      * Get API client
+     *
      * @return \Swagger\Client\ApiClient get the API client
      */
     public function getApiClient()
     {
         return $this->apiClient;
     }
-  
+
     /**
      * Set the API client
+     *
      * @param \Swagger\Client\ApiClient $apiClient set the API client
+     *
      * @return TCSApi
      */
-    public function setApiClient(ApiClient $apiClient)
+    public function setApiClient(\Swagger\Client\ApiClient $apiClient)
     {
         $this->apiClient = $apiClient;
         return $this;
     }
-  
-    
+
     /**
-     * getTemplateDetails
+     * Operation getTemplateDetails
      *
      * Provides all template types with providers information
      *
      * @param int $wiki_id Wikia ID (required)
      * @param int $page_id Article ID (required)
-     * @return \Swagger\Client\TemplateClassification\Storage\Models\TemplateTypeProvider[]
      * @throws \Swagger\Client\ApiException on non-2xx response
+     * @return \Swagger\Client\TemplateClassification\Storage\Models\TemplateTypeProvider[]
      */
     public function getTemplateDetails($wiki_id, $page_id)
     {
-        list($response, $statusCode, $httpHeader) = $this->getTemplateDetailsWithHttpInfo ($wiki_id, $page_id);
-        return $response; 
+        list($response) = $this->getTemplateDetailsWithHttpInfo($wiki_id, $page_id);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Immediately return this expression instead of assigning it to the temporary variable "$response". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1488)